src/content/glossary/index.md

```diff
@@ -92,6 +92,12 @@ Decentralized application. At a minimum, it is a [smart contract](#smart-contrac
 
 See [non-fungible token (NFT)](#nft)
 
+### defi {#defi}
+
+Short for "decentralized finance," a broad category of [Dapps](#dapp) aiming to provide financial services backed by the blockchain, without any intermediaries, so anyone with an internet connection can participate.
+
+<a href="/en/dapps/#how-to-use-an-application-built-on-ethereum">More about Dapp uses<Emoji text=":point_right:" size={1} marginLeft={0.75} /></a>
```

Comment:
```suggestion
[More about Dapp uses <Emoji text=":point_right:" size={1} marginLeft={0.75} />](/dapps/#how-to-use-an-application-built-on-ethereum)
```

Reply:
We can & should continue to use markdown syntax where possible.

Note: we can & should also drop the language path (/en/ in this case). When we use markdown links, Gatsby uses [our Link component](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Link.js) under the hood, which automatically adds the language path. This makes it easier when translating pages - we don't have to update the links everywhere.

Reply:
@samajammin 👍 Will definitely remove the /en/ language path. (Note though that all `static.js` pages will interpret the anchor tags into the custom `Link` component as well, not just when using markdown... custom component in the works will do the same):
![image](https://user-images.githubusercontent.com/54227730/98171841-3dc9bb00-1ea5-11eb-8862-9a07cc69b193.png)

I'll fix these now to get this pulled in first.